ledger/ledger-api-test-tool-on-canton/canton-test-runner.sh

```diff
@@ -18,9 +18,15 @@ PARTICIPANT_1_LEDGER_API_PORT=5011
 PARTICIPANT_1_MONITORING_PORT=7000
 PARTICIPANT_2_HOST=localhost
 PARTICIPANT_2_LEDGER_API_PORT=5021
+PARTICIPANT_3_HOST=localhost
+PARTICIPANT_3_LEDGER_API_PORT=5031
+PARTICIPANT_4_HOST=localhost
+PARTICIPANT_4_LEDGER_API_PORT=5041
```

Comment:
Note we don't have any test involving more than two participants and by default the test tool will pick the participants in the same order as they are presented. You can use `--shuffle-participants` to make sure other participants are picked. This only ensures (to a certain degree, given the randomness) that all participants are used as part of the tests, note that currently there is no test that uses more than two participants in a given test.

Reply:
I noticed one lonely `TransactionServiceIT.TXTreeBlinding` test with a third `delta` participant. I had been contemplating `--shuffle-participants`, but was a bit concerned about introducing flakiness. In a separate PR, and perhaps in the Canton repo first, I will give this a try.